instance/gandi.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Gandi DNS - Helper dunctions
+"""
+
+# Imports #####################################################################
+
+import xmlrpc.client
+
+from django.conf import settings
+from django.core.cache import cache
+
+
+# Logging #####################################################################
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+# Classes #####################################################################
+
+class GandiAPI():
+    """
+    Gandi API proxy object
+    """
+    def __init__(self):
+        self.client = xmlrpc.client.ServerProxy('https://rpc.gandi.net/xmlrpc/')
```

Comment:
@antoviaque nit: hardcoded url - maybe it should be a django setting? Or even an `__init__` parameter?